tests/test_core.py

```diff
@@ -8072,6 +8072,14 @@ def test_pthread_create(self):
     self.set_setting('EXIT_RUNTIME')
     self.do_run_in_out_file_test('tests', 'core', 'pthread', 'create.cpp')
 
+  @node_pthreads
+  def test_pthread_create_2(self):
+    self.set_setting('PTHREAD_POOL_SIZE', '8')
+    self.set_setting('EXIT_RUNTIME')
+    if not self.has_changed_setting('INITIAL_MEMORY'):
+      self.set_setting('INITIAL_MEMORY', '64mb')
+    self.do_runf(path_from_root('tests', 'pthread', 'test_pthread_create.cpp'), '')
```

Comment:
I wonder if running these tests under `tests/pthread` that normally run in `test_browser.py` is something we should try to do more consistently here in `test_core.py`?   Is there a particular reason you include this one as part of this PR?

Reply:
The reason I included `tests/pthread/test_pthread_create.cpp` in `test_core` was that in #9763 it was mentioned that this could not be run with Node, this PR seems to fix that. IIRC there was a PR that enabled the `test_browser` tests to run with Node, so this seems a bit redundant.

Removed this with commit d0f3bc0d316abf34afca1c95f65baec91d22179d.

Reply:
Sorry I think didn't mean to make you remove those two tests necessarily, I was mostly wondering out loud.     For `tests/pthread/test_pthread_create.cpp` I think it probably is worth trying to get all of the `tests/pthread/` tests working in test_core.py.. perhaps as a followup?   You could include this single one with at TODO to add the rest if you like.  Or not, its up to you.